tools/release/extract-release-notes.ts

```diff
@@ -10,8 +10,11 @@ export function extractReleaseNotes(changelogPath: string, versionName: string)
   // section of a version by starting with the release header which can either use the markdown
   // "h1" or "h2" syntax. The end of the section will be matched by just looking for the first
   // subsequent release header.
-  const releaseNotesRegex = new RegExp(`(##? ${escapedVersion}.*?)##? 7\\.`, 's');
+  const releaseNotesRegex = new RegExp(`(##? ${escapedVersion} "(.*?)" \\(.*?)##? \\d\\.\\d`, 's');
```

Comment:
Should this be `\d+`?

Reply:
I was thinking that too, but since we use `new RegExp` we need to double escape this.

```
new RegExp('\d') --> /d/
```

Reply:
I mean specifically the `+`. We'll eventually get to v10

Reply:
Ah good point. Fixed. thanks!

<sup>For some reason I didn't realize there is a plus 🤦‍♂️</sup>